packages/web/src/common/store/premiumContent/sagas.ts

```diff
@@ -293,47 +294,48 @@ function* updateCollectibleGatedTrackAccess(
   }
 }
 
-const PREMIUM_TRACK_POLL_FREQUENCY = 2000
+const PREMIUM_TRACK_POLL_FREQUENCY = 5000
```

Comment:
i see that it takes much longer than 2 seconds, so decreasing the frequency here to reduce number of requests to DNs